open_city_profile/tests/full_stack_tests.py

```diff
@@ -1,6 +1,7 @@
 import uuid
```

Comment:
Now that this file gets renamed anyway, could there be a better name for it? As there are no tests in this file, the _tests suffix is a bit misleading.

Also is "full stack" the best term? Could it be "integration" or "component integration"? Or maybe it could be left out altogether.

So something like "graphql_conf", "graphql_call_settings", "graphql_testing_helpers" could be more helpful names? What do you think?

Reply:
Agree. Maybe "graphql_test_helpers" would be a nice choice.

Reply:
Changed to `graphql_test_helpers` in 529a0ca.